site/src/pages/CliAuthPage/CliAuthPageView.tsx

```diff
@@ -21,11 +21,9 @@ export const CliAuthPageView: FC<CliAuthPageViewProps> = ({ sessionToken }) => {
       <Welcome>Session token</Welcome>
 
       <p css={styles.instructions}>
-        Copy the session token below and{" "}
-        <strong css={{ whiteSpace: "nowrap" }}>
-          paste it in your terminal
-        </strong>
```

Comment:
This had a risk of causing text rendering issues if the user shrunk their viewport too much